handlers/update_task.go

```diff
@@ -0,0 +1,46 @@
+package handlers
+
+import (
+	"time"
+
+	"github.com/go-openapi/runtime/middleware"
+
+	runtime "github.com/AliShan302/Task_Schedular"
+	domainErr "github.com/AliShan302/Task_Schedular/errors"
+	"github.com/AliShan302/Task_Schedular/gen/restapi/operations"
+	"github.com/AliShan302/Task_Schedular/models"
+)
+
+// NewUpdateTask handles request for updating task
+func NewUpdateTask(rt *runtime.Runtime) operations.UpdateTaskHandler {
+	return &updateTask{rt: rt}
```

Comment:
there should be ctx Context here.

Like this:
`
func NewUpdateTask(ctx context.Context, rt *runtime.Runtime) operations.UpdateTaskHandler {
	return &updateTask{
		ctx: ctx,
		rt:  rt,
	}
}
`